src/util.js

```diff
@@ -63,17 +98,15 @@ export function styleObjToCss(s) {
 				prop[0] == '-'
 					? prop
 					: JS_TO_CSS[prop] ||
-					  (JS_TO_CSS[prop] = prop.replace(CSS_REGEX, '-$1').toLowerCase());
+					  (JS_TO_CSS[prop] = prop.replace(CSS_REGEX, '-$&').toLowerCase());
 
 			let suffix = ';';
-			let isNumber = typeof val === 'number';
-			if (isNumber && SUFFIX_CACHE[name]) {
-				suffix = 'px;';
-			} else if (
-				isNumber &&
-				IS_NON_DIMENSIONAL.test(prop.toLowerCase()) === false
+			if (
+				typeof val === 'number' &&
+				// Exclude custom-attributes
+				!name.startsWith('--') &&
```

Comment:
Is this check doing the same thing as line 98, i.e. are they both trying to check for custom properties? Do we get any perf benefit by consolidating them?